src/components/ClickCopy.vue

```diff
@@ -25,6 +25,26 @@ export default {
   props: {
     gradient: String,
     name: String,
+    vanilla: Boolean,
+  },
+  data() {
+    return {
+      mounted: false,
+    }
+  },
+  mounted() {
+    this.mounted = true
+  },
+  computed: {
+    value() {
+      if (!this.mounted) return null
+      
+      if (this.vanilla) {
+        return getComputedStyle(this.$refs.gradientUnderline).getPropertyValue('background-image')
+      }
+
+      return this.gradient
+    }
```

Comment:
All this logic in the components that's sole purpose is to copy a value to the clipboard feels overkill.

I also think if we were to add this feature, I would want it separate from the current button style and position, so a new component is an option.